### ISSUE TITLE
Demo, spaces were added to the texts

### ISSUE THREAD

**Issue by syn0419** [AILab-CVC/YOLO-World#341]
Why were spaces added when loading text during the demo, but not added during deployment?
Demo：
![1716435012724](https://github.com/AILab-CVC/YOLO-World/assets/49259869/6cdf91f2-104e-47b7-b554-e4f71a8ed62b)

Deploy：
![1716435038735](https://github.com/AILab-CVC/YOLO-World/assets/49259869/624b3e3e-2873-47d0-934c-41edaa482b94)

**Comment by wondervictor** [comment]
Please check the latest `export_onnx.py`, we have added an `--add-padding` option.

**Comment by syn0419** [comment]
> Please check the latest `export_onnx.py`, we have added an `--add-padding` option.
Thank you for the reply. I'd like to ask why it's necessary to deliberately add spaces to the text during inference.

**Comment by wondervictor** [comment]
No, it's not necessary and depends on whether adding the padding performs well or not. It will be fixed.

**Comment by demooooooo0303** [comment]
> No, it's not necessary and depends on whether adding the padding performs well or not. It will be fixed.

所以推理的时候text到底加不加空格，我发现区别会很大
